packages/native/src/components/MapView/MapMarker.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { View, Text, StyleSheet } from "react-native";
+import { LatLng, MarkerProps } from "react-native-maps";
+import { MapMarkerProps } from "./types";
+import { Marker as WebMarker } from "@react-google-maps/api";
+import MapCallout from "./MapCallout";
+
+interface IMarkerContext {
+  pinColor?: string | undefined;
+  calloutOpened: boolean;
+  position: LatLng | undefined;
+  toggleCallout: (value: boolean) => void;
+}
+
+export const markerContext = React.createContext<IMarkerContext>({
+  pinColor: "#E74C3C",
+  calloutOpened: false,
+  position: undefined,
+  toggleCallout: () => {},
+});
+
+const MapMarker: React.FC<MapMarkerProps> = ({
+  pinColor,
+  latitude,
+  longitude,
+  title,
+  description,
+  children,
+}) => {
+  const { Provider } = markerContext;
+  const [calloutOpened, toggleCallout] = React.useState(false);
+  const [marker, setMarker] = React.useState<MarkerProps | null>(null);
+  const handleMarkerClick = () => toggleCallout(true);
+  const handleOnLoad = (m: MarkerProps) => setMarker(m);
+  let mappedChildren;
+  if (!children) {
+    if (title || description) {
+      mappedChildren = (
+        <MapCallout showTooltip anchor={marker}>
+          <View style={style.tooltip}>
+            {title && <Text style={style.title}>{title}</Text>}
+            {description && (
+              <Text style={style.description}>{description}</Text>
+            )}
+          </View>
+        </MapCallout>
+      );
+    }
+  } else {
+    mappedChildren = React.Children.map(children, (child, index) => {
+      return React.cloneElement(child as React.ReactElement, {
+        index,
+        anchor: marker,
+      });
+    });
+  }
+  // const mappedChildren = React.Children.map(children, (child, index) => {
+  //   return React.cloneElement(child as React.ReactElement, {
+  //     index,
+  //     anchor: marker,
+  //   });
+  // });
```

Comment:
Can we either uncomment this or remove? I'm assuming remove because it looks to be the same as the block above

Reply:
done